lib/content._coffee

```diff
@@ -6,21 +6,18 @@ redis = require 'redis'
 URL = require 'url'
 Worker = require './worker'
 
+pkgcloud = require('pkgcloud')
+cloudClient = pkgcloud.storage.createClient
```

Comment:
I figure we might as well only do this once up here, though I'm open to other locations, of course.